.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: "CI/CD for main"
+
+env:
+  INFURA_API_KEY: ${{ secrets.INFURA_KEY_FOR_CI_CD }}
+  MNEMONIC: ${{ secrets.MNEMONIC_FOR_CI_CD }}
+  PRIVATE_KEY: ${{ secrets.ETH_PRIVATE_KEY_FOR_CI_CD }}
```

Comment:
We should probably create a fresh set of these for each repo we set up actions on.

Reply:
I was wondering the same thing, but it's easier to expose them to a new repo that needs them then to create new ones each time. And in terms of security, someone could steal that shared mnemonic, as long as it's only used there I think it's fine.